python/requirements.txt

```diff
@@ -8,6 +8,7 @@ pytest==6.2.1
 pytest-cov==2.11.0
 pytest-html==3.1.1
 PyYAML==5.4.1
+scikit-learn==0.21.3
```

Comment:
as mentioned, let's not add a huge new dependency just for a test.

Reply:
Yes, we can just add it to the `build-requirements.txt` I think.